config/environments/development.rb

```diff
@@ -53,4 +53,6 @@
   config.colorize_logging = true
 
   config.action_controller.allow_forgery_protection = true
+
+  config.api_url = "http://localhost:3000"
```

Comment:
3000 is typically the UI worker port, not the API worker one.

I would suggest using [this](https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_web_service_worker.rb#L6) if anything.

Of course this could change if you're using something like Foreman to run the workers, as we allow the port for each worker to be set using an environment variable.

Reply:
@carbonin Would each developer be changing the development.rb (akin to Gemfile.dev.rb). This just gives you a default setting that should work for most developers. For some one running in MIQ_SPARTAN mode they would have to change it to http://localhost:4000. If someone want to test with HTTPS they could change that to https://localhost:4443.